src/handlers/ui-events.test.js

```diff
@@ -29,42 +29,6 @@ beforeEach(() => {
   component = shallow(<Farmhand />)
 })
 
-describe('handleItemPurchaseClick', () => {
-  test('calls purchaseItem', () => {
-    jest
-      .spyOn(component.instance(), 'purchaseItem')
-      .mockImplementation(() => {})
-    handlers().handleItemPurchaseClick(testItem({ id: 'sample-item-1' }), 3)
-
-    expect(component.instance().purchaseItem).toHaveBeenCalledWith(
-      testItem({ id: 'sample-item-1' }),
-      3
-    )
-  })
-})
-
-describe('handleItemSellClick', () => {
-  test('calls sellItem', () => {
-    jest.spyOn(component.instance(), 'sellItem').mockImplementation(() => {})
-    handlers().handleItemSellClick(testItem({ id: 'sample-item-1' }))
-
-    expect(component.instance().sellItem).toHaveBeenCalledWith(
-      testItem({ id: 'sample-item-1' }),
-      1
-    )
-  })
-})
-
-describe('handleViewChange', () => {
-  beforeEach(() => {
-    handlers().handleViewChange({ target: { value: stageFocusType.SHOP } })
-  })
-
-  test('changes the view type', () => {
-    expect(component.state('stageFocus')).toEqual(stageFocusType.SHOP)
-  })
-})
```

Comment:
This is obviated by this more recent RTL-based test: https://github.com/jeremyckahn/farmhand/blob/f3371f6c96926bc67869eba7a77ae5f56b939619/src/shell/navigation.test.js#L8-L16